app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheetController.kt

```diff
@@ -119,9 +123,17 @@ class ScheduleAppointmentSheetController @Inject constructor(
   }
 
   private fun scheduleDefaultAppointmentDateForSheetCreates(events: Observable<UiEvent>): Observable<PotentialAppointmentDate> {
-    return events
-        .ofType<ScheduleAppointmentSheetCreated>()
-        .withLatestFrom(configProvider) { _, config -> config.defaultTimeToAppointment }
+    val protocolStream = currentFacilityStream()
+        .map { Optional.toOptional(it.protocolUuid) }
+        .filterAndUnwrapJust()
```

Comment:
The problem with this approach is that if we have users who update to this version from before the protocols were added to the `Facility` model, this event would never emit and the screen would not update with a date at all.

How does the screen behave in this case? Should we use the default value we were using earlier?

Reply:
![Screenshot_1573100430](https://user-images.githubusercontent.com/1978065/68359989-2c3fbd00-0144-11ea-9c2b-fe9c7b673382.png)

This is what the screen ends up looking like, and the `DONE` button doesn't do anything since there are no appointment dates that have been generated. We should handle this case, probably use the value from the config as a fallback.